frontend/packages/kubevirt-plugin/integration-tests/v2v/utils/v2v.utils.ts

```diff
@@ -0,0 +1,12 @@
+export async function readFile(filename: string) {
```

Comment:
This is not used, is it?
if not, please remove

Reply:
removed